### DISCUSSION TITLE
v2.0.0-beta.1

### DISCUSSION THREAD

**Discussion by ALTaleX531** [ALTaleX531/TranslucentFlyouts#33] [General]
1. Unify and optimize the reading logic of the registry, the current CONFIG.md is not completely fit for this version.
2. Fix a lot of rendering issues.
3. Fix the issue that custom rendering sometines cannot work.
4. Add a experimental registry value: "EnableFluentAnimation" at TranslucentFlyouts\Menu\, set it to 1 to activate.

<hr /><em>This discussion was created from the release <a href='https://github.com/ALTaleX531/TranslucentFlyouts/releases/tag/'>v2.0.0-beta.1</a>.</em>

**Comment by ghost** [comment]
Glad you are polising this tweak which is already my daily driver.
Noticed the rounded corners have changed to sharp corners in this new update by default.
How do we get the rounded corners?
Is this the value to be changed? there is nothing on my registry so I have to create a custom value
CornerType
# DWORD (32-bit) value that controls the corner type
# 0: Keep the default, use small corners
# 1: Square corners
# 2: Large corners
# 3: Small corners
# Only applicable to Windows 11

Also I tried the EnableFluentAnimation but I can understand ~~it is not that smooth right now as it flashes the full menu size while animating.~~ Does changing to DIrect-2d in registry make it smoother?

**Reply by ALTaleX531** [reply]
Actually I didn't change it to sharp corners in this new update by default which I think it was a bug, can you show me your OS build information? The CornerType value does control the corner of the popup menu but I think it is also affected by this bug... 
About the EnableFluentAnimation, I wonder what it looks like in your OS, can you post a video of it?

**Reply by ghost** [reply]
The left is the new updated flyout and right one is the older which has rounded corner.
![Screenshot 2023-08-06 190730](https://github.com/ALTaleX531/TranslucentFlyouts/assets/100816109/811f1910-51c1-439a-af9b-890815707077)
Also I am not able to replicate that animating issue i had recently as it is working well right now. It is although a bit unsettling the way it animates but here is the video

https://github.com/ALTaleX531/TranslucentFlyouts/assets/100816109/a1a8f894-8e03-4bb2-a67b-a3634a8edb95

**Reply by ALTaleX531** [reply]
Actually it use small round corners in the new release to respect default popup menu corners option, you can manually change it, it is not a bug.
I think the latest release has solved the animation issue.

**Reply by ghost** [reply]
I also think its not a bug. it could be the symbol file you used for this update from windows 10!
